frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -130,14 +140,23 @@ export const SecurityBreakdownPopup: React.FC<SecurityBreakdownPopupProps> = ({
                         v.metadata.name
                       }`}
                     >
-                      {pluralize(
-                        resource.filter(({ metadata }) => metadata.name === v.metadata.name).length,
-                        'namespace',
-                      )}
+                      {namespace
+                        ? imageNameClamped(
+                            resource.find(({ metadata }) => metadata.name === v.metadata.name).spec
```

Comment:
nit: we need to consider the case if find returns undefined

Reply:
i figured there shouldn't be a case where `find` returns undefined since `v` is an element from `fixableVulns` which is a subset of `resource`, and there is a check for `fixableVulns` being empty which prevents execution when `fixableVulns` is empty

Reply:
i figured there shouldn't be a case where `find` returns undefined since `v` is an element from `fixableVulns` which is a subset of `resource`, and there is a check for `fixableVulns` being empty which prevents execution when `fixableVulns` is empty